config/locales/devise.en.yml

```diff
@@ -0,0 +1,62 @@
+en:
```

Comment:
Please could you merge this file into config/locales/en.yml? 

Now that we've got everything into one yaml file, we should keep that consistent (it makes Josh's life easier, too). Everything is under en:devise, so it should be pretty trivial to do.